backend/pkg/logger/logger.go

```diff
@@ -0,0 +1,80 @@
+package logger
+
+import (
+	"fmt"
+	"sync"
+	"sync/atomic"
+
+	"github.com/HyperloopUPV-H8/h9-backend/pkg/abstraction"
+)
+
+const (
+	dataLogger abstraction.LoggerName = "data"
+)
+
+type Logger struct {
+	running        *atomic.Bool
+	subloggersLock sync.RWMutex
+	subloggers     map[abstraction.LoggerName]abstraction.Logger
+}
+
+func (logger *Logger) Start(startKeys []abstraction.LoggerName) {
+	logger.subloggersLock.Lock()
+	defer logger.subloggersLock.Unlock()
+
+	if logger.running.Load() {
+		fmt.Printf("Logger already running")
+		return
+	}
+
+	for _, name := range startKeys {
+		if sublogger, ok := logger.subloggers[name]; ok {
+			go sublogger.Start(nil)
+		}
+	}
+
+	logger.running.Store(true)
+	fmt.Printf("Logger started")
+}
+
+func (logger *Logger) PushRecord(record abstraction.LoggerRecord) error {
+	loggerChecked, exists := logger.subloggers[record.Name()]
+	if exists {
+		loggerChecked.PushRecord(record)
+		return nil
+	}
+	return ErrLoggerNotFound{record.Name()}
```

Comment:
```suggestion
	sublogger, ok := logger.subloggers[record.Name()]
	if !ok {
	    return ErrLoggerNotFound{Name: record.Name()}
	}
	return sublogger.PushRecord(record)
```
the `ok` variable name for the second parameter is an idiom amongst go developer, and I also like to use it, this way we keep the code base with a consistent style (of course if you must use a different name)